pkg/certsuite/certsuite.go

```diff
@@ -14,6 +21,16 @@ import (
 	"github.com/test-network-function/cnf-certification-test/pkg/provider"
 )
 
+func LoadChecksDB() {
+	accesscontrol.LoadChecks()
+	certification.LoadChecks()
+	lifecycle.LoadChecks()
+	manageability.LoadChecks()
+	networking.LoadChecks()
+	observability.LoadChecks()
+	performance.LoadChecks()
+}
+
```

Comment:
@jmontesi Could the `LoadChecks` changes be in a separate PR?  I love the idea of getting rid of those Init() funcs and it doesn't look like its tied to the logging changes.

Reply:
Sure, here's the [PR](https://github.com/test-network-function/cnf-certification-test/pull/1664).